apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx

```diff
@@ -99,7 +102,7 @@ export function TransactionSubmittedContent({
         <>
           {!isProgressBarSetup && <styledEl.Title>{getTitleStatus(activityDerivedState)}</styledEl.Title>}
           {showSafeSigningInfo && <GnosisSafeTxDetails chainId={chainId} activityDerivedState={activityDerivedState} />}
-          <EthFlowStepper order={order} showProgressBar={!!showProgressBar && !cancellationFailed} />
+          {!cancellationFailed && !isFinished && <EthFlowStepper order={order} showProgressBar={!!showProgressBar} />}
```

Comment:
`cancellationFailed` is included in the `isFinished` boolean

```suggestion
          {!isFinished && <EthFlowStepper order={order} showProgressBar={!!showProgressBar} />}
```

Reply:
Thank you!